magprime/templates/static_views/super_swag_bag.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html>
+<head>
+  <title>Super Swag Bag</title>
+  <style type="text/css">
+    html, html * { font-family: Arial, Helvetica, sans-serif; }
+    img { max-width: 100%; }
+    h1 { text-align: center; }
+  </style>
+</head>
+<body>
+
+<h1>Super Swag Bag</h1>
+
+<p>
+  We've heard your feedback, and are making our supporter package swag bags
+  better than ever!
+</p>
+<p>
+  Are you tired of boring looking bags that are either empty, or filled with
+  stuff that is really boring. Well, boy, do we have a surprise for you. Our
+  theme changes every year, and with that new theme comes a lot of fun ways
+  to interact with it.
+</p>
+<p>
+  Our swag bags are filled with awesome themed goodies,
+  and are exclusively available to pregistrations only.
```

Comment:
This is another bit of text that's kinda confusingly worded. The swag bag isn't available to people just cause they preregister -- it's available to people who kick in enough for it.